### ISSUE TITLE
Teacher model's performance fluctuates on USPS dataset.

### ISSUE THREAD

**Issue by happen2me** [huawei-noah/Data-Efficient-Model-Compression#15]
I am trying to do an adaptation task with DAFL. I tried to train the LeNet5 on MNIST, but after I applied it to USPS dataset, a strange thing occurred: the performance of trained LeNet5 varied largely on USPS dataset when I trained LeNet5 with the same hyperparameters for several times, accuracy varied from 40% to 80%. Under this circumstance, it's hard to fix a baseline for adaptation.

It is a little off the topic as DAFL is designed for distillation, but can anyone give me a light on why this happens?

**Comment by happen2me** [comment]
It seems to be overfitting, I solved the problem by adding dropout layers to LeNet.
